working-with-multiple-clusters.md

```diff
@@ -0,0 +1,196 @@
+$ kubectl get svc
+```
+
+As you've noticed, kubectx and kubens help you work with different clusters and namespaces, which is a little less problematic.
+
+But what if there's even a better way for developers? Let's see.
+
+## Developing with Contexts in Okteto
+
+Okteto has support for configuring the context and namespace your applications might need. Hence, you have consistency at coding time because all developers are going to use the same configurations. Also, you'll have fewer commands to run. Even if the existing commands from kubectcl or the kubectx utility, you still need to switch manually. Conversely, with Otketo, you can include this configuration in the YAML manifest.
+
+For instance, following [our sample from Rust](https://okteto.com/blog/getting-started-with-okteto-and-rust/), you could modify the `okteto.yaml` to this:
+
+```
+name: hello-world
+image: okteto/rust:1
+command: ["bash"]
+workdir: /okteto
+context: docker-desktop
+securityContext:
+  capabilities:
+    add:
+    - SYS_PTRACE
+forward:
+  - 3000:3000
+  - 2345:2345
+persistentVolume:
+  enabled: false
+```
+
+Notice how we're specifying which context to use with the line `context: docker-desktop`. And even if you're currently pointing to the remote cluster context, Okteto will use the Kubernetes cluster from the Docker Desktop app. Additionally, if you don't want to have hardcoded values, you could use an environment variable, like `context: $APP_CONTEXT`. Developers can have different names for a local context, and they only need to set up `$APP_CONTEXT` environment variable so that Okteto points to the proper cluster.
+
+Let's give it a try, run the following command to force the usage of the remote cluster:
+
+```
+$ kubectx remote-cluster
+Switched to context "remote-cluster".
+```
+
+Now that the `okteto.yml` has the context configuration, run the `okteto up` command and compile the application, like this:
+
+```
+$ okteto up
+ ✓  Development container activated
+ ✓  Connected to your development container
+ ✓  Files synchronized
+    Context:   docker-desktop
+    Namespace: default
+    Name:      hello-world
+    Forward:   2345 -> 2345
+               3000 -> 3000
+
+Welcome to your development container. Happy coding!
+default:hello-world okteto> cargo run
+    Finished dev [unoptimized + debuginfo] target(s) in 0.78s
+     Running `target/debug/hello-world`
+Listening on http://0.0.0.0:3000
+```
+
+Notice how the Okteto output tells you which context is using.
+
+You should now be able to see the application running in the local cluster, like this:
+
+```
+$ curl localhost:3000
+Hello Kubernetes!
+```
+
+When you close the Okteto terminal, notice that you're still using the remote cluster context. Locally, nothing changed because Okteto allowed knows which context to use to deploy your application where it should be running. At least, you're going to make sure you're developing in a local cluster and not in a production cluster.
```

Comment:
```suggestion
When you close the Okteto terminal, notice that you're still using the remote cluster context. Locally, nothing changed because Okteto automatically knows which context to use to deploy your application where it should be running. With this, you can rest assured that you're developing in a local cluster and not in a production cluster!
```